### ISSUE TITLE
FROM ARGS are not checked

### ISSUE THREAD

**Issue by mxbossard** [hadolint/hadolint#1012]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
With supplied Dockerfile : 
ARG foo bar mean nothing, there is no equal sign. The linter should complain.
FROM is referencing a missing ${baz} ARG. The linter should complain.
ARG pif paf mean nothing, there is no equal sign. The linter should complain.

### Actual behavior
The linter do not complain.

### Steps to reproduce the behavior

**Dockerfile** (if relevant)
```Dockerfile
ARG foo bar
FROM alpine${foo}:${baz}
ARG pif paf
```

**Comment by mxbossard** [comment]
I tested ARG foo bar on Docker and it works like if we declared 2 args foo and bar on 2 separate lines.
But this behavior is not documented, so should the linter support it ?
